### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_ColorContrast : The color contrast ratio of the control border when focus goes on the "Learn more about permission " link is less than 3:1

### ISSUE THREAD

**Issue by prasadaps** [microsoft/AzureStorageExplorer#3457]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.11;#Win32;#A11ySev2;#Benchmark;#ColorContrast;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen reader: JAWS(2020.1912.11ILM), NVDA Version:2020.2

### Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Blob gets uploaded, right click on blob and select "Get Shared Access Signature" control and hit enter.
7. Navigate to the link "Learn more about permission" and check the color contrast ratio of selected link bored.

### Actual:
The color contrast ratio of the control border when focus goes on the "Learn more about permission " link is less than 3:1.

### Expected:
The color contrast ratio of the select control border should be greater than 3:1.

### User Impact:
Low vision users may not able to see the control has keyboard focus.

### Attachment
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prphat_microsoft_com/EXSW8UbqARlKgL6uGah2btgB4N3ny6P5vft3O0yHZIoU0A?e=N8qfy6

**Comment by MRayermannMSFT** [comment]
![image](https://user-images.githubusercontent.com/21112697/96175274-84984680-0edf-11eb-9751-f132d1ec768a.png)

**Comment by prasadaps** [comment]
Verified the issue in latest version of azure tool storage explorer application. the issue is not reproducing  so closing the bug. please refer the closing attachment. https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prphat_microsoft_com/EQvUDWQfyjVCqzGBKUdMB88BsRWYJYdMFDsLXK-ePV_Ffw?e=8arUh3

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
